### ISSUE TITLE
Add overall trade history

### ISSUE THREAD

**Issue by ghost** [Leasehold/lisk-dex-ui#33]
Adding a quick overview of trade history, for example last 20 trades, gives more insight in the activity of the DEX.

**Comment by jondubois** [comment]
Good idea. I'm thinking maybe it may be better to just show the user's own trade history. Something to think about.
